packages/astro/src/core/build/static-build.ts

```diff
@@ -198,9 +204,12 @@ interface GeneratePathOptions {
 }
 
 async function generatePath(pathname: string, opts: StaticBuildOptions, gopts: GeneratePathOptions) {
-	const { astroConfig, logging, origin, routeCache } = opts;
+	const { astroConfig, logging, origin, pageNames, routeCache } = opts;
 	const { Component, internals, linkIds, pageData } = gopts;
 
+	// This adds the page name to the array so it can be shown as part of stats.
+  addPageName(pathname, opts);
```

Comment:
It looks like prettier/eslint did not catch this. I can’t recall if these are auto-fixed after merge.

Reply:
We disabled prettier to run in CI a while back because it was breaking releases (it would format in `main` after merging the release PR). We should probably add it back and have it only run in non-main branches.

Reply:
https://github.com/withastro/astro/commit/356d81abcb11abdc76d7e27ed6fc637cd98ffdde

Reply:
I ran `yarn format` and pushed the changes here. It's odd that EditorConfig didn't correct this.